client/src/main.js

```diff
@@ -23,19 +22,18 @@ function transferFolder(folder) {
     }
 }
 
-
 function render(template, target, context) {
     var html = nunjucks.render('src/templates/' + template + '.njk', context);
     fs.writeFileSync('dist/' + target + '.html', html)
 }
 
-function renderFolder(folder) {
+function renderFolder(folder, template) {
```

Comment:
Allow a custom template to be used (required for stats). The custom template will still only be passed `title` and `body`.